### PR TITLE
Add fixes for table overflow and image size

### PR DESCRIPTION
**Pull request by Eshan05**
Address issue #113, also see video in comments if it's up to your liking (I wasn't sure what image size would be viable). Made minimal changes only where needed however due to the auto-format it would look the whole file is changed so to be specific

- Change `nav.TOC` and `figure>p>img` in the media query 
- Add CSS class for `.table`

### CODE REVIEW

**Comment by Eshan05** [comment]
@jeremylt See if it's fine


https://github.com/user-attachments/assets/7a120434-cc70-4759-bf6f-756e034c4ef8

File: rules/style.css
Comment (jeremylt) [inline rules/style.css:187]:
```suggestion
}

```
small style tweak

File: rules/style.css
Comment (jeremylt) [inline rules/style.css:132]: Just so I can get a bit smarter on CSS - why the scale property?

**Comment by jeremylt** [comment]
This looks much nicer, thank you!

**Comment by jeremylt** [comment]
Note: CI won't run on this, but once you add that small fix I'll merge manually

**Review by jeremylt — APPROVED** [review]

File: rules/style.css
Comment (Eshan05) [reply rules/style.css:132]: Some images appeared quite small so to counter that

**Comment by Eshan05** [comment]
> Note: CI won't run on this, but once you add that small fix I'll merge manually

I don't follow, what small fix?

File: rules/style.css
Comment (jeremylt) [reply rules/style.css:187]: This newline here is the 'small fix'

Comment (Eshan05) [reply rules/style.css:187]: Oh I see, but you edited already. I have enabled "Edits by maintainers" so it should be done already

**Comment by jeremylt** [comment]
Thanks for these fixes!

**Comment by Eshan05** [comment]
> Thanks for these fixes!

No problem! Anytime

**Pull request merged**
